ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -51,6 +51,19 @@ images:
         | tar -xz -C /usr/local/bin
   from: src
   to: test-bin
+operator:
+  bundles:
+  - as: operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  - as: hub-operator-bundle
+    dockerfile_path: bundle-hub.Dockerfile
+  substitutions:
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-operator:latest
+    with: pipeline:kernel-module-management-operator
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-operator-hub:latest
+    with: pipeline:kernel-module-management-operator-hub
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-signimage:latest
+    with: pipeline:kernel-module-management-signimage
```

Comment:
Do we need them in the new branch as well? If we do we need a new file for it in addition to that one.

Reply:
Yes we do. Let us do that in a separate PR?

Reply:
Sure.